dockers/docker-lldp-sv2/start.sh

```diff
@@ -2,6 +2,11 @@
 
 sonic-cfggen -d -t /usr/share/sonic/templates/lldpd.conf.j2 > /etc/lldpd.conf
 
+# update the hostname. Not using device metadata as in multi-npu platform each namespace
+# has its own device metadata hostname. Using the hostname set by docker using host enviroment
```

Comment:
why the DEVCIE_METADATA['localhost']['hostname'] cannot not be the real hostname? The container is not a host, I think every asic should have its own name, but we do not necessarily call it hostname name. i grep the current code, it seems there are multiple places referring to the hostname. I do not know if we really want to change it.

```
dockers/docker-fpm-frr/frr/common/daemons.common.conf.j2:hostname {{ DEVICE_METADATA['localhost']['hostname'] }}
dockers/docker-fpm-gobgp/zebra.conf.j2:hostname {{ DEVICE_METADATA['localhost']['hostname'] }}
dockers/docker-fpm-quagga/bgpd.conf.j2:hostname {{ DEVICE_METADATA['localhost']['hostname'] }}
dockers/docker-fpm-quagga/zebra.conf.j2:hostname {{ DEVICE_METADATA['localhost']['hostname'] }}
dockers/docker-lldp-sv2/lldpd.conf.j2:configure system hostname {{ DEVICE_METADATA['localhost']['hostname'] }}
```

Reply:
@lguohan Looking into if we can change minigraph parsing to populate hostname for each asic also and use some other key for asic name in device metadata.

Reply:
@lguohan updated minigraph parsing to have hostname for each asic/namespace as system hostname.